### PR TITLE
[CI] Skip lib for xpu binary unit test

### PR DESCRIPTION
**Pull request by chuanqi129**
Skip .so and .a libraries under build/bin/ for test_xpu_bin in CI

Works for #114850 also

### CODE REVIEW

**Review by malfet — APPROVED** [review]
Sure, why not, but what is "UT"?

File: .ci/pytorch/test.sh [hunk 1]
```diff
@@ -692,7 +692,7 @@ test_xpu_bin(){
 
   for xpu_case in "${BUILD_BIN_DIR}"/*{xpu,sycl}*
   do
```
Comment (malfet) [inline .ci/pytorch/test.sh:694]: Please note, no other tests in this file have `do` on a newline
```suggestion
  for xpu_case in "${BUILD_BIN_DIR}"/*{xpu,sycl}*; do
```

**Comment by chuanqi129** [comment]
> Sure, why not, but what is "UT"?

Thanks @malfet for the review. Renamed the PR.

**Pull request closed without merging**